src/components/donation/Steps/Tip/Tip.tsx

```diff
@@ -0,0 +1,195 @@
+import Image from "components/Image/Image";
+import { humanize } from "helpers";
+import { useState } from "react";
+import { useController, useForm } from "react-hook-form";
+import { schema, stringNumber } from "schemas/shape";
+import { TipStep, setStep, setTip } from "slices/donation";
+import { useSetter } from "store/accessors";
+import BackBtn from "../common/BackBtn";
+
+const DEFAULT_PCT = "0.17";
+
+interface ITip {
+  pct: string;
+  amount: string;
+}
+
+type FV = {
+  tip: ITip;
+};
+
+const tipSchema = schema<ITip>({
+  amount: stringNumber(
+    (s) => s,
+    (n) => n.min(0, "can't be negative")
+  ),
+});
+
+const shape = schema<FV>({
+  tip: tipSchema,
+});
+
+export default function Tip({
+  details,
+  tip: persistedTip,
+  format = "pct",
+}: TipStep) {
+  const dispatch = useSetter();
+
+  const [symbol, amount, decimals = 2] = (() => {
+    switch (details.method) {
+      case "stripe":
+        return [details.currency.code, +details.amount];
+      case "daf":
+        return ["usd", +details.amount];
+      case "stocks":
+        return [details.symbol, details.numShares];
+      case "crypto":
+        return [details.token.symbol, +details.token.amount, 4];
+    }
+  })();
+
+  const initial: ITip = {
+    amount: `${amount * +DEFAULT_PCT}`,
+    pct: DEFAULT_PCT,
+  };
+
+  const {
+    control,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<FV>({
+    resolver: yupResolver(shape),
+    defaultValues: {
+      tip: persistedTip
+        ? {
+            amount: persistedTip.toString(),
+            pct: `${persistedTip / amount}`,
+          }
+        : initial,
+    },
+  });
+  const {
+    field: { value: tip, onChange: onTipChange },
+  } = useController<FV, "tip">({ name: "tip", control });
+
+  //if user selects custom, can't go back to %
+  const [isPct, setIsPct] = useState(format === "pct");
+
+  return (
+    <form
+      onSubmit={handleSubmit((v) =>
+        dispatch(
+          setTip({
+            tip: Number(v.tip.amount),
+            format: isPct ? "pct" : "amount",
+          })
+        )
+      )}
+      className="grid content-start p-4 @md:p-8"
+    >
+      <BackBtn type="button" onClick={() => dispatch(setStep("splits"))} />
+      <h4 className="mt-4 text-lg">
+        Choose a Donation for{" "}
+        <Image src={bgIcon} className="inline-block size-5" /> Better.giving
+      </h4>
+      <p className="text-gray-d1">
+        We are completely free, and rely on donations
+      </p>
+
+      {isPct && (
+        <Slider.Root
+          min={0}
+          max={1}
+          step={0.01}
+          value={[Number(tip.pct)]}
+          onValueChange={([pct]) =>
+            onTipChange({
+              amount: humanize(amount * pct, decimals),
+              pct,
+            })
+          }
+          className="relative flex items-center select-none touch-none mt-16"
+        >
+          <Slider.Track className="relative grow rounded-full h-1.5 bg-[#EAECEB]">
```

Comment:
I also believed this is a default thing to do, but read from [ux collective](https://uxdesign.cc/buttons-shouldnt-have-a-hand-cursor-part-2-4a6e1c8423a5) otherwise. But perhaps its too late to correct things as almost all of our clickable elements have `cursor-pointer` in it @SovereignAndrey

Reply:
That was an interesting read! Never considered the history of links or button's visual differences from links regarding needing a pointer change. 👍 

Leave as is for now. It is not a huge issue and we can sync all buttons one way or the other in a follow up PR.

Reply:
Interesting, will read up more about it.